demos/demos-v2.yaml

```diff
@@ -147,6 +147,25 @@ demos:
       cpu: 6800m
       memory: 15822Mi
       pvc: 28Gi
+  trino-subsea-data:
+    description: TODO
+    documentation: TODO
+    stackableStack: trino-superset-s3
+    labels:
+      - trino
+      - superset
+      - minio
+      - s3
+      - parquet
+    manifests:
+      - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/feat/marispace-demo/demos/trino-subsea-data/load-test-data.yaml
+      - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/feat/marispace-demo/demos/trino-subsea-data/create-table-in-trino.yaml
+      - plainYaml: https://raw.githubusercontent.com/stackabletech/demos/feat/marispace-demo/demos/trino-subsea-data/setup-superset.yaml
```

Comment:
Reminder to change links

Reply:
https://github.com/stackabletech/demos/pull/88/commits/2ece0adffa9f874b5eed5654827c1d478bab356b